lib/src/cli.rs

```diff
@@ -512,6 +515,27 @@ async fn usroverlay() -> Result<()> {
         .into());
 }
 
+/// Implementation of `bootc build commit`
+/// async fn lint() -> Result<()> {
+fn lint() -> Result<()> {
+    // TODO: Check if 1 or more folder under the /lib/modules/ directory
+    println!("Checking if we are in the container");
+    if ostree_ext::container_utils::is_ostree_container()? {
+        let modules_path = cap_std::fs::Dir::open_ambient_dir("lib/modules", cap_std::ambient_authority())?;
+        for entry in modules_path.entries()? {
```

Comment:
Also I forgot until looking that https://docs.rs/ostree-ext/latest/ostree_ext/bootabletree/fn.find_kernel_dir_fs.html exists...so we could just call that here, which is probably better as it ensure we use the exact same logic ostree is using.

Reply:
Wow this made our lives easy thank you!